kratos/geometries/geometry.h

```diff
@@ -647,9 +647,120 @@ class Geometry
     ///@name Operations
     ///@{
 
-    virtual Pointer Create( PointsArrayType const& ThisPoints ) const
+    /**
+     * @brief Creates a new geometry pointer
+     * @param rThisPoints the nodes of the new geometry
+     * @return Pointer to the new geometry
+     */
+    virtual Pointer Create(
```

Comment:
I think that we don't need to have this one virtual. In fact, it would be better to centralize the automatic creation of the Id in the geometry base class.

Reply:
So why you keep this virtual?